### PR TITLE
Feature 1833/hr click reset onboardee button on edit popup

### PR DESCRIPTION
**Pull request by MisheelSodgerel**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (S78901) [inline package.json:1]: Delete this file. This appears to be a mistake.

File: yarn.lock [hunk 2]
```diff
@@ -2,3 +2,36 @@
 # yarn lockfile v1
 
 
+"@babel/runtime@^7.7.6":
```
Comment (S78901) [inline yarn.lock:5]: Delete this file too. This appears to be a mistake.

File: web-ui/src/pages/OnboardProgressPage.jsx [hunk 3]
```diff
@@ -130,12 +130,9 @@ export default function OnboardProgressPage(onboardee) {
   ];
 
   async function submitInfo(onboardee) {
-    console.log(
-      "Submit info"
-    )
+    console.log("Submit info");
```
Comment (S78901) [inline web-ui/src/pages/OnboardProgressPage.jsx:133]: We can remove the consoles.

File: web-ui/src/index.js [hunk 4]
```diff
@@ -3,10 +3,12 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
+import { BrowserRouter as Router } from "react-router-dom";
 
 ReactDOM.render(
-    <App />, 
-  document.getElementById("root")
+  <Router>
```
Comment (S78901) [inline web-ui/src/index.js:9]: We don't need this at the root because it's already used in App.js. Delete this addition of Router.

**Review by S78901 — APPROVED** [review]
Great work!

**Review by S78901 — APPROVED** [review]

**Review by BenVuong — APPROVED** [review]

**Pull request merged**